pylinac/core/hillreg.py

```diff
@@ -0,0 +1,46 @@
+"""Perform non-linear regression using a Hill function"""
```

Comment:
I'm not familiar with the Hull function. Is this better for inflection points than the second derivative? Can you point me to some resources?

Reply:
I'm working from the Dutch NCS Report 33 - Linac QA: Beam Parameters for Non-Wedge High-Energy Photon Beams (attached). The preprint was provided to me by a colleague Theo van Soest author of Bistromath. The Netherlands Commission on Radiation Dosimetry is generally quite ahead of the curve when it comes to reports and position papers. The sigmoid (Hill) model is much more resistant to noise than the second derivative.
[Prepublication_-_NCS_Report_33_Beam_parameters_V2020-07-29.pdf](https://github.com/jrkerns/pylinac/files/5993489/Prepublication_-_NCS_Report_33_Beam_parameters_V2020-07-29.pdf)

Reply:
The beauty of this model is of course that it is very simple to add an edge parameter using the second derivative and then compare them all.